.changeset/chilled-beans-retire.md

```diff
@@ -0,0 +1,9 @@
+---
+"@kaizen/components": minor
+---
+
+# FilterBar
```

Comment:
Check where in the heading hierarchy this goes in the changelog, I think it's like 2 or 3

Reply:
Was totally wrong lol
```suggestion
#### FilterBar
```